win/sassc.vcxproj.filters

```diff
@@ -1,361 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is it an accidental removal? This file was added to categorize files in Solution Explorer.

Reply:
No, I removed it since it puts quite a burden on devs to keep it in sync.
And it didn't seem to be relevant for successful builds, so I removed it.
Is it possible to have only one master list of files for MSVC and still support this?

Reply:
Filter is just a logical view, a design-time artifact to list files in custom structure in the Visual Studio - https://stackoverflow.com/questions/6941868/why-do-vcxproj-filters-files-exist

Master:

![image](https://user-images.githubusercontent.com/3840695/61722930-d6e9c680-ad73-11e9-9de8-755a79c95c2f.png)

Branch - without these filters, the solution explorer looks like this:

![image](https://user-images.githubusercontent.com/3840695/61723000-fa147600-ad73-11e9-9d6e-b3101622721d.png)

this is an annoyance of vcxproj project system, that they try to keep the view separate from build. In other project systems of VS, they simply use filesystem hierarchy (with wildcard) as the default and add ability to link out-of-the-tree files that projects on top of filesystem view. Insult to the injury, vcxproj.filters do not support wildcards.

If we toggle to the filesystem view of project provided by VS, then we do not see LibSass files with out default structure (we recommend in docs to clone `sassc` at `libsass` root):

![image](https://user-images.githubusercontent.com/3840695/61723717-47ddae00-ad75-11e9-981c-7cef6e475ffd.png)

If the maintenance is becoming a nightmare (i.e. we are adding/moving files in libsass too often), lets remove this file. Otherwise we can keep it up to date with a little automation (I can add a task which will update the filter file and we can run it like `msbuild /t:UpdateFilters win/sassc.vcxproj`).

Reply:
Thanks for that thorough explanation @am11!
Need to think about how to proceed with this later, for the moment the files can stay in.
I will update the PRs for libsass/sassc in a few hours when I'm home. The most irritating thing
for me was actually that we had libsass source files referenced in sassc. Maybe there is at least
a way to get this also partially read from the actual libsass source directly (as with target file)?